homeassistant/util/unit_conversion.py

```diff
@@ -138,6 +144,15 @@ def get_unit_ratio(cls, from_unit: str | None, to_unit: str | None) -> float:
         from_ratio, to_ratio = cls._get_from_to_ratio(from_unit, to_unit)
         return from_ratio / to_ratio
 
+    @classmethod
+    @lru_cache
+    def _is_unit_inverses(cls, from_unit: str | None, to_unit: str | None) -> bool:
```

Comment:
```suggestion
    def _are_unit_inverses(cls, from_unit: str | None, to_unit: str | None) -> bool:
```
either `_is_unit_inverse` or `_are_unit_inverses`

Reply:
Fixed in dee7f070a28f28da7ceadceeb165d7c6753c81a4